src/main/kotlin/it/pagopa/helpdeskcommands/controllers/CommandsController.kt

```diff
@@ -13,10 +13,19 @@ class CommandsController() : CommandsApi {
     private val logger = LoggerFactory.getLogger(this.javaClass)
 
     override fun refundOperation(
+        xUserId: String?,
```

Comment:
```suggestion
        xUserId: String,
```

Reply:
make also the below parameter as non-nullable too

Reply:
Done here: dd050b01